x/launch/types/message_request_add_validator.go

```diff
@@ -10,14 +10,16 @@ const TypeMsgRequestAddValidator = "request_add_validator"
 var _ sdk.Msg = &MsgRequestAddValidator{}
 
 func NewMsgRequestAddValidator(
-	valAddress string,
+	creator string,
 	launchID uint64,
+	valAddress string,
```

Comment:
```suggestion
	creator,
	valAddress string,
	launchID uint64,
```

Reply:
I wanted to keep the format we have for all messages (signer -> id -> content)

I actually realized we didn't have it for `MsgRequestRemoveAccount` and `MsgRequestRemoveValidator` so I changed it for those